filter/index.js

```diff
@@ -1,19 +1,31 @@
+// function onlyEven(array) {
+//   const evens = [];
+//   for (let i = 0; i < array.length; i++) {
+//     if (array[i] % 2 === 0) {
+//       evens.push(array[i]);
+//     }
+//   }
+//   return evens;
+// }
+
+// refactor with filter!! OMG!
 function onlyEven(array) {
-  const evens = [];
-  for (let i = 0; i < array.length; i++) {
-    if (array[i] % 2 === 0) {
-      evens.push(array[i]);
-    }
-  }
-  return evens;
+  var evens = array.filter(function(even){
+    return (even % 2 === 0)
+  })
 }
-// refactor with filter!! OMG!
 
 function onlyOneWord(array) {
+  var single = array.filter(function(word) => {
+    return (word.length === 1)
+  })
```

Comment:
Similar to the comment above, when you use fat arrow you don't need the anonymous function and change your var to const!